x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/components/agent_enrollment_flyout/config_selection.tsx

```diff
@@ -64,6 +64,14 @@ export const EnrollmentStepAgentConfig: React.FC<Props> = (props) => {
   useEffect(
     function useDefaultConfigEffect() {
       if (agentConfigs && agentConfigs.length && !selectedState.agentConfigId) {
+        if (agentConfigs.length === 1) {
```

Comment:
is a 'length' a 0 based count?  should it be greater than 1 ?

I'm not strong with typescript / js, and so don't yet see how this changes the case where there are 2 configs.  If you tested it then thats great - just checking so I can understand the code base better.

Reply:
it should be greater than 1 one when we are in the scenario where the flyout is open not for a specific config, for a specific config we load the flyout with only one config.

Reply:
oh, ok - thanks much.  it looks great then.  :)